src/Test/StateMachine/Internal/Types.hs

```diff
@@ -18,6 +18,8 @@
 -- Stability   :  provisional
 -- Portability :  non-portable (GHC extensions)
 --
+-- This module defins
+--
```

Comment:
We should probably not encourage [defin](https://en.wikipedia.org/wiki/Shark_fin_soup#Demand_declines.2C_2005-present) of any kind.